libs/blocks/tabs/tabs.js

```diff
@@ -168,6 +243,16 @@ const init = (block) => {
     tabListItems[0].parentElement.remove();
   }
 
+  // Tab Paddles
+  const tabPaddles = createTag('div', { class: 'tab-paddles', role: 'scrollbar' });
+  const paddleLeft = createTag('button', { class: 'paddle paddle-left', disabled: '' });
```

Comment:
```suggestion
  const paddleLeft = createTag('button', { class: 'paddle paddle-left', disabled: '' }, LEFT_PADDLE);
```
You can likely simplify these and just insert the paddle element on create.